WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -565,6 +565,7 @@ public void onLoadData() {
                 } else {
                     switch (getPostListType()) {
                         case TAG_FOLLOWED:
+                            // fallthrough
```

Comment:
Good call @malinajirka , thanks for adding these comments. I think I personally like these type of comments to have a reference to what they are falling through to though, to help debug issues in the future if anyone changes the case order and that. In essence it would be cool if you could update the comments with that info.

It seems there's a similar example here https://github.com/wordpress-mobile/WordPress-Android/blob/4508f54aec0999175dab8225a759cc795332b91b/WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java#L365 too. WDYT?

Reply:
Good idea! ;) Fixed